oar/core/advisory_mgr.py

```diff
@@ -65,6 +66,52 @@ def change_ad_owners(self):
         except ErrataException as e:
             raise AdvisoryException("change advisory owner failed") from e
 
+    def check_greenwave_cvp_tests(self):
+        """
+        Check whether all the Greenwave CVP tests in all advisories
+        All the test status should be PASSED
+
+        Raises:
+            AdvisoryException: error found when checking CVP test result
+
+        Returns:
+            bool: True if all test are passed otherwise False
+        """
+        abnoraml_tests = []
+        try:
+            ads = self.get_advisories()
+            for ad in ads:
+                logger.info(
+                    f"checking Greenwave CVP test for advisory {ad.errata_id} ..."
+                )
+                tests = ad.get_greenwave_cvp_tests()
+                all_passed = True
+                if len(tests):
+                    for t in tests:
+                        status = t["attributes"]["status"]
+                        logger.debug(f"Greenwave CVP test {t['id']} status is {status}")
+                        valid_status = [CVP_TEST_STATUS_PASSED, CVP_TEST_STATUS_PENDING]
+                        if status not in valid_status:
+                            all_passed = False
+                            logger.error(
+                                f"Greenwave CVP test {t['id']} status is not {valid_status}"
+                            )
+                            abnoraml_tests.append(t)
+                    logger.info(
+                        f"Greenwave CVP tests in advisory {ad.errata_id} are {'all' if all_passed else 'not all'} passed"
+                    )
+                else:
+                    logger.info(
+                        f"advisory {ad.errata_id} does not have Greenwave CVP tests"
+                    )
+        except ErrataException as e:
+            raise AdvisoryException("Get greenwave cvp test failed") from e
+
+        if len(abnoraml_tests):
+            logger.error(f"NOT all Greenwave CVP tests are passed")
+
+        return abnoraml_tests
```

Comment:
Naybe we can descript the return with: return "these Greenwave CVP tests are failed:" +abnormal_tests